components/TorchboxAcademy/AcademyDayCards/AcademyDayCards.tsx

```diff
@@ -0,0 +1,53 @@
+import { Document } from '@contentful/rich-text-types';
+import RichText from 'components/RichText';
+import Button from 'components/Button';
+import MicrophonePerson from 'components/SVG/MicrophonePerson';
+import CoffeePerson from 'components/SVG/CoffeePerson';
+import { CharacterType } from 'types/Base';
+import styles from './AcademyDayCards.module.scss';
+
+type AcademyDay = {
+    title: string;
+    content: { json: Document };
+    link: string;
+    character: CharacterType;
+};
+
+const Card = ({ title, content, link, character }: AcademyDay) => (
+    <div className={styles.card}>
+        <div className={styles.icon}>
+            {character === 'MICROPHONE' ? (
+                <MicrophonePerson className={styles.microphonePerson} />
+            ) : null}
+            {character === 'COFFEE' ? (
+                <CoffeePerson className={styles.coffeePerson} />
+            ) : null}
+        </div>
+        <h3 className={styles.title}>{title}</h3>
+
+        <RichText theme="INDIGO" content={content} className={styles.content} />
+        <Button url={link} theme="WHITE" className={styles.button} internal>
```

Comment:
It would be good to modifer the endingChevron a bit to match the designs. Adding these attributes seems to work.
```
margin-left:
align-self: center;
```
<img width="259" alt="Screenshot 2022-03-31 at 09 23 21" src="https://user-images.githubusercontent.com/35934884/161010877-5d2640e0-d23e-407a-8bc5-a3755425fe29.png">
<img width="204" alt="Screenshot 2022-03-31 at 09 23 16" src="https://user-images.githubusercontent.com/35934884/161010879-b87a7b6e-8bd1-47d4-ab65-3d571fee7513.png">

Reply:
Thank you 👍